calculator-styles.css

```diff
@@ -0,0 +1,58 @@
+* {
+  font-family: sans-serif;
+}
+.calculator {
+  border: 2px solid;
+  border-radius: 10px;
+  background-color: #000000 ;
+  margin: 10px auto;
+  padding: 5px;
+  width: 300px;
+}
+.title {
+  margin: 20px auto;
+  width: 120px;
+}
+.buttons {
+  box-sizing: border-box;
+  margin: 5px;
+  padding: 5px;
+}
+.first-row, .second-row, .third-row, .fourth-row{
```

Comment:
You can define a class as `calculator-row` and apply it to this so you don't have to repeat yourself.